ultraLatency.py

```diff
@@ -41,8 +41,10 @@
                       help="Calibration Module is set to use voltage step pulsing instead of default current pulse injection", 
                       metavar="voltageStepPulse")
     parser.add_option("--vt2", type="int", dest="vt2",
-                      help="VThreshold2 DAC value for all VFATs (v2b electronics only)", metavar="vt2", default=0)
-    
+                      help="VThreshold2 DAC value for all VFATs (v2b electronics only)", metavar="vt2", default=0) 
+    parser.add_option("--chan", type="int", dest="chan",
+                      help="Channel on the VFATs on which to run the latency scan", metavar="chan", default=0)
+
```

Comment:
keep this in alphabetical order.

Reply:
Additionally the help menu needs to be updated to illustrate this is only relevant when calling with the `--internal` option.